조성원/오픈채팅방.js

```diff
@@ -0,0 +1,36 @@
+const solution = (record) => {
+  const answer = [];
+  const users = new Map();
+
+  for (const string of record) {
+    const data = string.split(" ");
+
+    switch (data[0]) {
+      case "Enter":
+        users.set(data[1], data[2]);
+        break;
+      case "Change":
+        users.set(data[1], data[2]);
+        break;
```

Comment:
```suggestion
      case "Enter":
      case "Change":
        users.set(data[1], data[2]);
        break;
```

같은 코드 중복이라, 각각 명시적으로 작성해주고 싶은게 아니라면 합쳐서 한 번에 처리해도 될 것 같아요!

Reply:
스위치 문 잘 안써서 몰랐는데 이렇게 써도 되는군요!! 저도 알아갑니다..!

Reply:
앗 사실 예전에 풀어놨던 코드인데 이제와서보니 왜 저렇게 작성했나 싶긴 하네요 ㅋㅋㅋㅋ
감사합니다!!